catkin_virtualenv/src/catkin_virtualenv/venv.py

```diff
@@ -65,8 +65,8 @@ def initialize(self, python, use_system_packages, extra_pip_args, clean=True):
             raise RuntimeError(error_msg)
 
         preinstall = [
-            "pip==20.1",
-            "pip-tools==5.1.2",
+            "pip==21.3.1",
```

Comment:
does this support python2 and python3, or just the latter?

if we're going to drop python2 support entirely, probably worth updating the README

Reply:
This version just support python3.6 to 3.10. Yep will update the read-me